Translated/Story/04-05 DOLLS on Mt. Magnus/msq_011131.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
You seem to have accidentally blanked this file out.

Reply:
how the fuck

Reply:
And moved it to the wrong place, now that I look at it.

Reply:
okay fixed